src/libraries/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.WebAssembly.cs

```diff
@@ -0,0 +1,241 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Instead of manually implementing the PNSE-stubbed-out version, could we get tooling to do it for us? e.g.
https://github.com/dotnet/runtime/blob/a0014a04e6dfd99e42f1c4a11efc8bc239d4faa9/src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj#L13

Reply:
Learn something new every day ;-).